information-schema/information-schema-metrics-summary.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: METRICS_SUMMARY
 summary: Learn the `METRICS_SUMMARY` system table.
-aliases: ['/docs/dev/system-tables/system-table-metrics-summary/','/docs/dev/reference/system-databases/metrics-summary/']
+aliases: ['/docs/dev/system-tables/system-table-metrics-summary/','/docs/dev/reference/system-databases/metrics-summary/',''/tidb/dev/system-table-metrics-summary/'']
```

Comment:
```suggestion
aliases: ['/docs/dev/system-tables/system-table-metrics-summary/','/docs/dev/reference/system-databases/metrics-summary/','/tidb/dev/system-table-metrics-summary/']
```
@nullnotnil Please help fix this alias in another PR.

Reply:
I've already fixed it in release-4.0 branch in the cherry-picked PR https://github.com/pingcap/docs/pull/3353.

Reply:
Fixed the alias in https://github.com/pingcap/docs/pull/3367. FYI @nullnotnil @TomShawn